wren-ai-service/pyproject.toml

```diff
@@ -19,6 +19,7 @@ numpy = "==1.26.4"
 sqlparse = "==0.4.4"
 sqlglot = "==22.5.0"
 orjson = "==3.10.3"
+sf-hamilton = {extras = ["visualization"], version = "^1.63.0"}
```

Comment:
please remember to use the fixed version in the future.

Reply:
thanks for the reminder. this PR was published for testing purposes, so I think it still has lots of details that need to be modified. including this one. 😸